vpcclassicv1/vpc_classic_v1.go

```diff
@@ -20099,23 +20208,11 @@ type NetworkACLRulePrototype struct {
 	Name *string `json:"name,omitempty"`
 
 	// The protocol to enforce.
-	Protocol *string `json:"protocol,omitempty"`
+	Protocol *string `json:"protocol" validate:"required"`
 
 	// The source IP address or CIDR block.  The CIDR block `0.0.0.0/0` applies to all addresses.
 	Source *string `json:"source" validate:"required"`
 
-	// The date and time that the rule was created.
-	CreatedAt *strfmt.DateTime `json:"created_at,omitempty"`
-
-	// The URL for this Network ACL rule.
-	Href *string `json:"href,omitempty"`
-
-	// The unique identifier for this Network ACL rule.
-	ID *string `json:"id,omitempty"`
-
-	// The IP version for this rule.
-	IPVersion *string `json:"ip_version,omitempty"`
-
```

Comment:
More breaking changes.  These might be passable in prerelease but they will create major problems once the SDK goes to 1.0.